### DISCUSSION TITLE
Not sure if the problem in shadow db

### DISCUSSION THREAD

**Discussion by elfpileggi** [prisma/prisma#14347] [Q&A]
Error:
![image](https://user-images.githubusercontent.com/22692516/179429877-15d91058-a562-4e96-ae2c-6a3e9cbd0fad.png)

Shcema
![image](https://user-images.githubusercontent.com/22692516/179429914-b1da7d73-f475-49e2-a2c3-75a68f5a0487.png)

Migration SQL
![image](https://user-images.githubusercontent.com/22692516/179430361-ca5a6a30-5c36-49ac-8d51-7cd05471df91.png)

Using an existing database I created a new table and related it to a "legacy" table. It seems to me that the migration is not copying my database correctly to the shadow db, because when I create a script to insert data in the new table give the error of fk problem, but if I add a if statement in the begining of the script to fill data into the legacy table works perfectly.

Workaround
![image](https://user-images.githubusercontent.com/22692516/179430544-1443e99e-f9e0-4bff-b3c3-18ec6961e3b9.png)

because when I create a script to insert data in the new table it gives the foreign key problem error, but if I add an if statement at the beginning of the script to fill the data in the legacy table, it works perfectly.

Could this be some kind of problem with the shadow database?

**Comment by ludralph** [comment]
Hi @elfpileggi 👋 

Are you still experiencing this issue? 

Could you please provide a minimal reproduction so I can reproduce this locally and get back to you on it?

**Reply by vladi-stevanovic** [reply]
👋 Hello @elfpileggi  I'm going through pending GitHub Discussions to check if you need any help. Is this issue still open?
Please let us know if we can investigate it further or if you've already solved it, we'd love to know how!

It would be super useful for other users facing this problem! 🙌

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
